File: historian-core/src/test/java/net/yslibrary/historian/Cursors.java
package net.yslibrary.historian;

import android.database.Cursor;

/**
 * Utility methods for {@link android.database.Cursor}
 */

public class Cursors {

  private Cursors() {
    // no-op
  }

  public static String getString(Cursor cursor, String column) {
    return cursor.getString(cursor.getColumnIndex(column));
  }
}

File: historian-core/src/test/java/net/yslibrary/historian/Cursors.kt
package net.yslibrary.historian

import android.database.Cursor

/**
 * Utility methods for [Cursor]
 */
object Cursors {
    @JvmStatic
    fun getString(cursor: Cursor, column: String): String =
        cursor.getString(cursor.getColumnIndexOrThrow(column))
}
